Pi2BoardPkg/Drivers/DisplayDxe/DisplayDxe.c

```diff
@@ -293,11 +339,12 @@ DisplayBlt(
         break;
 
     default:
-        ASSERT_EFI_ERROR(EFI_SUCCESS);
         break;
     }
 
-    return EFI_SUCCESS;
+Exit:
+    ASSERT_EFI_ERROR(Status);
```

Comment:
I'd be careful about adding the assert here.  You could cause issues with people running debug builds.